js/api.js

```diff
@@ -1,16 +1,54 @@
-var Database, NULL, RegisterExtensionFunctions, Runtime, SQLite, Statement, apiTemp, i, sqlite3_bind_blob, sqlite3_bind_double, sqlite3_bind_int, sqlite3_bind_parameter_index, sqlite3_bind_text, sqlite3_changes, sqlite3_clear_bindings, sqlite3_close_v2, sqlite3_column_blob, sqlite3_column_bytes, sqlite3_column_double, sqlite3_column_name, sqlite3_column_text, sqlite3_column_type, sqlite3_create_function_v2, sqlite3_data_count, sqlite3_errmsg, sqlite3_exec, sqlite3_finalize, sqlite3_free, sqlite3_open, sqlite3_prepare_v2, sqlite3_prepare_v2_sqlptr, sqlite3_reset, sqlite3_result_double, sqlite3_result_null, sqlite3_result_text, sqlite3_step, sqlite3_value_blob, sqlite3_value_bytes, sqlite3_value_double, sqlite3_value_int, sqlite3_value_text, sqlite3_value_type;
-
-SQLite = {};
+var Database, RegisterExtensionFunctions, Runtime, Statement, apiTemp, i, sqlite3_bind_blob, sqlite3_bind_double, sqlite3_bind_int, sqlite3_bind_parameter_index, sqlite3_bind_text, sqlite3_changes, sqlite3_clear_bindings, sqlite3_close_v2, sqlite3_column_blob, sqlite3_column_bytes, sqlite3_column_double, sqlite3_column_name, sqlite3_column_text, sqlite3_column_type, sqlite3_create_function_v2, sqlite3_data_count, sqlite3_errmsg, sqlite3_exec, sqlite3_finalize, sqlite3_free, sqlite3_open, sqlite3_prepare_v2, sqlite3_prepare_v2_sqlptr, sqlite3_reset, sqlite3_result_double, sqlite3_result_null, sqlite3_result_text, sqlite3_step, sqlite3_value_blob, sqlite3_value_bytes, sqlite3_value_double, sqlite3_value_int, sqlite3_value_text, sqlite3_value_type;
+
+// Constants
+var NULL = 0;
+var SQLite = {};
+SQLite.OK = 0;
+SQLite.ERROR = 1;
+SQLite.INTERNAL = 2;
+SQLite.PERM = 3;
+SQLite.ABORT = 4;
+SQLite.BUSY = 5;
+SQLite.LOCKED = 6;
+SQLite.NOMEM = 7;
+SQLite.READONLY = 8;
+SQLite.INTERRUPT = 9;
+SQLite.IOERR = 10;
+SQLite.CORRUPT = 11;
+SQLite.NOTFOUND = 12;
+SQLite.FULL = 13;
+SQLite.CANTOPEN = 14;
+SQLite.PROTOCOL = 15;
+SQLite.EMPTY = 16;
+SQLite.SCHEMA = 17;
+SQLite.TOOBIG = 18;
+SQLite.CONSTRAINT = 19;
+SQLite.MISMATCH = 20;
+SQLite.MISUSE = 21;
+SQLite.NOLFS = 22;
+SQLite.AUTH = 23;
+SQLite.FORMAT = 24;
+SQLite.RANGE = 25;
+SQLite.NOTADB = 26;
+SQLite.NOTICE = 27;
+SQLite.WARNING = 28;
+SQLite.ROW = 100;
+SQLite.DONE = 101;
+SQLite.INTEGER = 1;
+SQLite.FLOAT = 2;
+SQLite.TEXT = 3;
+SQLite.BLOB = 4;
+SQLite.NULL = 5;
+SQLite.UTF8 = 1;
+
+var SQL = {}
 
 Module['onRuntimeInitialized'] = function() {
-  // Generated by CoffeeScript 1.12.4
+    console.log('ok')
```

Comment:
Why did you add a `console.log` ? Libraries should have undocumented side-effects, like writing to the console.